pytorch_lightning/trainer/trainer.py

```diff
@@ -2471,6 +2474,39 @@ def _active_loop(self) -> Optional[Union[FitLoop, EvaluationLoop, PredictionLoop
     Logging properties
     """
 
+    @property
+    def logger(self) -> Optional[LightningLoggerBase]:
+        return self._logger
+
+    @logger.setter
+    def logger(self, new_logger: Union[LightningLoggerBase, None]) -> None:
+        if new_logger:
+            if isinstance(new_logger, LoggerCollection):
+                self._logger = new_logger
+                self._loggers = [_logger for _logger in new_logger]
+            else:
+                self._logger = new_logger
+                self._loggers = [new_logger]
+        else:
+            self._logger = None
+            self._loggers = []
+
+    @property
+    def loggers(self) -> List[LightningLoggerBase]:
+        return self._loggers
+
+    @loggers.setter
+    def loggers(self, new_loggers: Union[Iterable[LightningLoggerBase], None]) -> None:
+        if new_loggers:
+            self._loggers = [_logger for _logger in new_loggers]
+            if len(self._loggers) == 1:
+                self._logger = self._loggers[0]
+            else:
+                self._logger = LoggerCollection(self._loggers)
```

Comment:
```suggestion
            self._logger = self._loggers[0] if len(self._loggers) == 1 else LoggerCollection(self._loggers)
```
:nit. just 3 less lines :)